libs/libarchive/Makefile

```diff
@@ -78,6 +78,7 @@ CMAKE_OPTIONS += \
 	-DENABLE_ZSTD=OFF \
 	-DENABLE_LIBXML2=OFF \
 	-DENABLE_PCREPOSIX=OFF \
```

Comment:
We can drop this, right?

Reply:
why? Looks like it is still used: https://github.com/libarchive/libarchive/blob/v3.7.4/CMakeLists.txt#L225C1-L225C86

Reply:
libpcre is gone I think

Reply:
let me know if I need to adapt this or not :)